src/main/scala/cromwell/engine/CallExecutionActor.scala

```diff
@@ -55,19 +54,12 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
    * Schedule work according to the schedule of the `backoff`.
    */
   private def scheduleWork(work: => Unit): Unit = {
-    val interval = backoff.nextBackOffMillis().millis
-    context.system.scheduler.scheduleOnce(interval) {
+    val interval = backendCall.pollBackoff.next.backoffMillis
+    context.system.scheduler.scheduleOnce(interval.millis) {
       work
     }
```

Comment:
Yeah that's what I was thinking too - but switching to FSM just to mutate a backoff object sounded overkill.
I imagine we'll want to switch to FSM for this CEA too eventually anyway though so this might as well be the occasion.